packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -1,26 +1,34 @@
 /**
  * @typedef {import('appeals-service-api').Api.AppealCaseWithAppellant} AppealCaseWithAppellant
+ * @typedef {import('appeals-service-api').Api.AppealSubmission} AppealSubmission
  */
 
 /**
  * @typedef DueDocumentType
  * @type {object}
  * @property {string} deadline the date by which the document is due
- * @property {number} dueInDays the number of days remaining until the deadline expires
+ * @property {number} [dueInDays] the number of days remaining until the deadline expires
  * @property {string} documentDue the type of document which is due next
- * @property {string} baseUrl the base url for the document type
+ * @property {string} [baseUrl] the base url for the document type
  */
 
 const { calculateDueInDays } = require('./calculate-due-in-days');
 
-const { APPEAL_STATE, DECISION_OUTCOME } = require('@pins/business-rules/src/constants');
+const { DECISION_OUTCOME } = require('@pins/business-rules/src/constants');
 const { getAppealTypeName } = require('./full-appeal/map-planning-application');
+const { businessRulesDeadline } = require('./calculate-deadline');
 
 const questionnaireBaseUrl = '/manage-appeals/questionnaire';
 const statementBaseUrl = '/manage-appeals/appeal-statement';
 const finalCommentBaseUrl = '/manage-appeals/appeal-comment';
 const proofsBaseUrl = '/manage-appeals/proofs-of-evidence';
 
+// MAP DATABASE RETURN OBJECTS TO DASHBOARD DISPLAY DATA
+
+/**
+ * @param {AppealCaseWithAppellant} appealCaseData
+ */
```

Comment:
Ideally we'd have a type for the "display data" so these functions could have  a meaningful return value.

Reply:
Yes, it's definitely something we want.  I will add and if necessary this can be further refined in the dashboard consolidation work